File: app/src/main/java/trunk/doi/base/bean/CollectionBean.java
package trunk.doi.base.bean;

import android.os.Parcel;
import android.os.Parcelable;


/**
 * Author: Othershe
 * Time: 2016/8/12 14:34
 */


public class CollectionBean {



    private String _id;
    private String desc;
    private String dataTime;
    private String url;



    public String get_id() {
        return _id;
    }

    public void set_id(String _id) {
        this._id = _id;
    }

    public String getDesc() {
        return desc;
    }

    public void setDesc(String desc) {
        this.desc = desc;
    }

    public String getDataTime() {
        return dataTime;
    }

    public void setDataTime(String dataTime) {
        this.dataTime = dataTime;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    @Override
    public String toString() {
        return "CollectionBean{" +
                "_id='" + _id + '\'' +
                ", desc='" + desc + '\'' +
                ", dataTime='" + dataTime + '\'' +
                ", url='" + url + '\'' +
                '}';
    }
}

File: app/src/main/java/com/gas/app/bean/CollectionBean.kt
package com.gas.app.bean

/**
 *
 */
class CollectionBean {
    private var _id: String? = null
    var desc: String? = null
    var dataTime: String? = null
    var url: String? = null
    fun get_id(): String? {
        return _id
    }

    fun set_id(_id: String?) {
        this._id = _id
    }

    override fun toString(): String {
        return "CollectionBean{" +
                "_id='" + _id + '\'' +
                ", desc='" + desc + '\'' +
                ", dataTime='" + dataTime + '\'' +
                ", url='" + url + '\'' +
                '}'
    }
}